src/js/view/view.js

```diff
@@ -278,15 +278,22 @@ define([
                 '.jw-thumb:after'
             ];
 
-            var iconHovers = ['.jw-icon:hover'];
+            var iconHovers = ['.jw-icon:hover', '.jw-option:hover'];
             var sliders = ['.jw-progress'];
             var containers = ['.jw-container'];
 
+
             addStyle('color', iconHovers, _model.get('skinColorActive'));
             addStyle('color', icons, _model.get('skinColorInactive'));
             addStyle('background', cues, _model.get('skinColorInactive'));
             addStyle('background', sliders, _model.get('skinColorActive'));
             addStyle('background', containers, _model.get('skinColorBackground'));
+
+            addStyle('color', ['.jw-active-option'], _model.get('skinColorActive'));
+            addStyle('background', ['.jw-active-option'], _model.get('skinColorInactive'));
+
+            addStyle('color', ['.jw-icon-hd', '.jw-icon-cc'], _model.get('skinColorActive'));
```

Comment:
We don't currently have a distinct "jw-on/jw-active" style for tooltip icons, since that's covered by our default styling.  Just targeting 'jw-icon-tooltip' instead of those two classes would introduce that logic on the volume tooltip as well.  This is logic targeted at our menu tooltips so maybe it makes sense to have that as a specific class.